containers/python-examples/PyTorch/docker-compose.yml

```diff
@@ -1,50 +1,17 @@
 version: "3.4"
 
 services:
-  server:
-    image: ghcr.io/deephaven/server-pytorch:${VERSION:-latest}
-    expose:
-      - '8080'
-    volumes:
-      - ./data:/data
-      - api-cache:/cache
-    environment:
-      - JAVA_TOOL_OPTIONS=-Xmx4g -Ddeephaven.console.type=python
+  deephaven:
+    image: ghcr.io/deephaven/server-pytorch:latest
     ports:
-      - 6006:6006
-
-  web:
-    image: ghcr.io/deephaven/web:${VERSION:-latest}
-    expose:
-      - '8080'
+      - "${DEEPHAVEN_PORT:-10000}:10000"
     volumes:
       - ./data:/data
-      - web-tmp:/tmp
-
-  grpc-proxy:
-    image: ghcr.io/deephaven/grpc-proxy:${VERSION:-latest}
     environment:
-      - BACKEND_ADDR=server:8080
-    depends_on:
-      - server
-    expose:
-      - '8080'
-
-  envoy:
-    image: ghcr.io/deephaven/envoy:${VERSION:-latest}
-    depends_on:
-      - web
-      - grpc-proxy
-      - server
-    ports:
-      - "${DEEPHAVEN_PORT:-10000}:10000"
+      - JAVA_OPTS=-Xmx4g
 
   examples:
-    image: ghcr.io/deephaven/examples:latest
+    image: ghcr.io/deephaven/examples
```

Comment:
We should keep the latest tag IMO instead of having it be implicit. (Other places as well)